collectors/node.go

```diff
@@ -125,21 +123,23 @@ func (l NodeLister) List() (v1.NodeList, error) {
 	return l()
 }
 
-func RegisterNodeCollector(registry prometheus.Registerer, kubeClient kubernetes.Interface, namespace string) {
+func RegisterNodeCollector(registry prometheus.Registerer, kubeClient kubernetes.Interface, namespaces []string) {
 	client := kubeClient.CoreV1().RESTClient()
 	glog.Infof("collect node with %s", client.APIVersion())
-	nlw := cache.NewListWatchFromClient(client, "nodes", metav1.NamespaceAll, fields.Everything())
-	ninf := cache.NewSharedInformer(nlw, &v1.Node{}, resyncPeriod)
+
+	ninfs := NewSharedInformerList(client, "nodes", []string{metav1.NamespaceAll}, &v1.Node{})
```

Comment:
I was not sure whether passing `[]string{metav1.NamespaceAll}` as a dummy namespace list is suitable here... shouln't there be some variable with such constant (it is used in several places - persistentvolumes). Also, should we iterate over the namespaces later?

Maybe matter of taste but I would still prefer to discuss it.

Reply:
I think this way is perfectly fine, we were using the same constant before after all.